send-pr

```diff
@@ -43,21 +43,18 @@ def parse_options
     opts.on("-l", "--list-PRs", "List pull requests") do |l|
       options[:list] = true
     end
-
-    opts.on("--access-token TOKEN", "Github access token") do |t|
-      options[:access_token] = t
-    end
   end.parse!
 
   raise OptionParser::ParseError, "missing --user" if options[:user].nil?
   raise OptionParser::ParseError, "missing --repository" if options[:repository].nil?
   raise OptionParser::ParseError, "missing --number" if options[:number].nil?
   raise OptionParser::ParseError, "missing --to" if options[:to].nil?
 
-  options[:access_token] = ENV['GITHUB_ACCESS_TOKEN'] if options[:access_token].nil?
+  options[:access_token] = ENV['GITHUB_TOKEN']
```

Comment:
Maybe I followed the lead of hub a little bit too closely :) Either version is fine with me.